forms-flow-api/src/formsflow_api/services/application.py

```diff
@@ -497,9 +498,11 @@ def fetch_task_variable_values(task_variable, form_data):
         if task_variable and form_data:
             task_keys = [val["key"] for val in task_variable]
             variables = {
-                key: {"value": json.dumps(form_data[key])}
-                if isinstance(form_data[key], (dict, list))
-                else {"value": form_data[key]}
+                key: (
```

Comment:
Just to confirm, the changes to push all variables to BPM is not part of this PR right ?

Reply:
When the "select all fields" option is enabled, all form fields are captured in the task_variable within the form_process_mapper. During form submission, the fields will be pushed to BPM based on the task_variable found in the mapper. So here all fields should be pushed to bpm